### ISSUE TITLE
Changing page slug erases unsaved content edits

### ISSUE THREAD

**Issue by sebastiangreger** [k-next/kirby#4102]
## Description
When changing the slug of a page while unsaved content is present, that content vanishes (though it can be restored by changing the slug back to the previous value).

**Expected behavior**  
Either changing the slug should not erase unsaved content edits, or (as a fallback expectation) unsaved content edits should never disappear without a warning.

## To reproduce
1. Create a new note on a Starterkit site
2. Add a new block (e.g. a headline)
3. Open "Settings > Change URL", edit the slug and submit with "Change" button

## Your setup
3.6.1.1

**Your system (please complete the following information)**
 - Device: PC
 - OS: Linux Mint
 - Browser: FF
 - Version: 96

**Comment by afbora** [comment]
This issue fixed in 3.6.2 https://github.com/getkirby/kirby/issues/4006 

Could you try on 3.6.2-rc.1 please? 
https://github.com/getkirby/kirby/releases/tag/3.6.2-rc.1

**Comment by sebastiangreger** [comment]
:see_no_evil: Oh my, and it was even me reporting that!

This would be very embarrassing if I hadn't first made a search for `is:issue sebastiangreger` in Issues, as I somehow had a hunch I wrote a ticket for that. It did not show up, so I assumed I only intended to write a ticket.

Thank you @afbora and apologies for the false alert – tried with 3.6.2-rc.1 and the bug is of course gone indeed :)

PS: If anybody can advise how to see all the issues I have authored in Github in one place, I'd be really grateful. This is not the first time I've been struggling to find an issue I have created somewhere. I'm sure there' a UI somewhere, as indeed searching for one's own username in Issues does not show them.
